pkg/ingress/group_loader.go

```diff
@@ -3,22 +3,20 @@ package ingress
 import (
 	"context"
 	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
 	"github.com/pkg/errors"
 	networking "k8s.io/api/networking/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/client-go/tools/record"
-	"regexp"
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/annotations"
 	"sigs.k8s.io/aws-load-balancer-controller/pkg/k8s"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sort"
-	"strings"
 )
 
 const (
 	defaultGroupOrder  int64 = 0
-	minGroupOrder      int64 = 1
```

Comment:
can we make it within range -1000~1000. I think this should sufficient for all use cases i can imaging.
We can extend this if there is any valid use case that need access beyond this range.

enforce limits on fields like this can avoid us introduce any backwards incompatible changes. (e.g. some might start set a order with max int64 and we later accidentally set it to int32 in some future release and break them)

Reply:
all rest code lgtm

Reply:
Done :+1: